### ISSUE TITLE
Black screen on Quest 2 and Quest 3 Browser

### ISSUE THREAD

**Issue by ghost** [De-Panther/unity-webxr-export#409]
**Describe the bug**
Built the sample desert scene, and trying to open it in the Quest browser. After the app loads, clicking VR to launch in VR just results in a black environment.

**To Reproduce**
1. Build sample desert scene
2. Open Quest browser and go to web page hosting the app
3. Click VR
4. App launches with black screen

**Expected behavior**
See the desert sample scene in VR.

**Screenshots**

**Unity info (please complete the following information):**
- Editor version: 6000.0.30f1
- WebXR package version: 0.22.1
- WebXR Interactions version: 0.22.1-preview
- Git or OpenUPM: Git
- Using old components (like the Desert sample) or Input System + XR Interaction Toolkit: 
- Built-in render pipeline or URP:

**Headset (please complete the following information):**
 - Device: Meta Quest 2 and Meta Quest 3
 - Browser: Meta Quest Browser

**Comment by De-Panther** [comment]
Missing info.
URP or Built-in RP?
The URP sample or the Built-in RP sample?

**Comment by ghost** [comment]
URP

**Comment by De-Panther** [comment]
using gamma color space or linear?

**Comment by ghost** [comment]
It is set to gamma

**Comment by De-Panther** [comment]
There should be a warning saying to use linear. Please try that.

**Comment by ghost** [comment]
I switched it to linear, which got rid of the warnings, but it is still a black screen.

**Comment by De-Panther** [comment]
Interesting. Thanks.
Can you please share your project? (both the unity project and build)
I can't reproduce.

**Comment by ghost** [comment]
I can't share the project, I will try to reproduce in a new empty project.

**Comment by De-Panther** [comment]
Thanks.
Another thing that might be an issue, is the URP asset.
You can try to use the WebXR URP renderer asset instead of the default URP asset.
I think the issue might be the MSAA

**Comment by ghost** [comment]
Disabled MSAA on my URP asset and it worked!

**Comment by leventalpsal** [comment]
I've experienced something similar. On Unity 6.2.9 Meta XR SDKv81 when I disabled Renderer Compatibility Mode in URP settings (which is the right thing to do) I was getting black screen on builds, not even the Unity splash screen was displayed. This was only on builds, not in editor or XR simulator.

For me the solution was to : 

XR Plug-in Management => OpenXR => "Foveated Rendering API" setting should be "SRP Foveation", but it was set to Legacy, which was causing the issue.
